go-ethereum-on-ubuntu/azuredeploy.json

```diff
@@ -36,10 +36,10 @@
         "Standard_A2",
         "Standard_A3",
         "Standard_A4",
-        "Standard_D1",
-        "Standard_D2",
-        "Standard_D3",
-        "Standard_D4"
+        "Standard_D1_v2",
```

Comment:
We want to avoid this pattern of putting VM sizes in parameters - it creates a maintenance problem.  Let's pick a size that works for the scenario and leave it as a var.